### PR TITLE
Add Artisense RD00 board files

### PR DESCRIPTION
**Pull request by m-byte**
VID/PID as assigned by Espressif (see https://github.com/espressif/usb-pids)

### CODE REVIEW

File: ports/esp32s2/boards/artisense_rd00/board.c [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 Scott Shawcroft for Adafruit Industries
```
Comment (jepler) [inline ports/esp32s2/boards/artisense_rd00/board.c:6]: Please update these files with your own copyright information.  E.g., "Copyright (c) 2021 <your name>", adding "for <company name>" if applicable.

File: ports/esp32s2/boards/artisense_rd00/mpconfigboard.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Scott Shawcroft for Adafruit Industries
```
Comment (jepler) [inline ports/esp32s2/boards/artisense_rd00/mpconfigboard.h:6]: here as well

**Review by jepler — CHANGES_REQUESTED** [review]
Thank you!  I saw some small items to correct.

**Review by jepler — APPROVED** [review]
Thank you!  This looks OK to merge if the CI passes.

**Comment by dhalbert** [comment]
@m-byte Hi - This board doesn't look like it's been added to circuitpython.org yet. If I'm wrong, sorry, otherwise please submit a PR to https://github.com/adafruit/circuitpython-org. If you are not going to produce this board, let us know, thanks.

**Pull request merged**